src/utils.jl

```diff
@@ -416,3 +416,20 @@ recursive_copy(gs::GenericStore) = GenericStore(recursive_copy(gs.name),
                                                 recursive_copy(gs.typ),
                                                 gs.doc,
                                                 gs.exported)
+
+function todo_path(x::ModuleStore)
+    for (_, a) in x.vals
+        todo_path(a)
+    end
+end
+
+function todo_path(x::Union{DataTypeStore,FunctionStore})
+    for m in x.methods
+        todo_path(m)
+    end
+end
+function todo_path(x::MethodStore)
+    # @info x.file
+end
+function todo_path(x)
```

Comment:
Unclear?

Reply:
This stuff can be ignored for now - when we're downloading a cache from the central server we need to scrub  paths to make them relative to the install path of the package, and then link it to the actual install location on the users machine (somehow).